src/app/admin/_lib/consts.js

```diff
@@ -9,8 +9,13 @@ export const RESOURCES = {
   event: 'event',
   eventTag: 'eventTag',
   faq: 'faq',
+  organization: 'organization',
 };
 
+export const ORGANIZATION_TYPES = ['Психологічний центр', 'Лікарня', 'Соціальна служба'];
+export const DISTRICTS = ['Личаківський', 'Залізничний', 'Шевченківський', 'Галицький', 'Сихівський', 'Франківський'];
+export const THERAPIES = ['Індивідуальна', 'Для дітей і підлітків', 'Сімейна', 'Групова', 'Для пар', 'Для бізнесу'];
```

Comment:
Instead of using consts for storing prisma generated enums.

Reply:
These fields are not prisma generated enums